### PR TITLE
Use automatic version numbers

### PR DESCRIPTION
**Pull request by costateixeira**
Just learned that the template does that automatically, so we don't need to maintain the version on each artifact

### CODE REVIEW

**Comment by costateixeira** [comment]
@ritikarawlani you can merge. don't mind the repo name change in the readme, it will be updated automatically

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <!--badges-->
 [![License: CC BY-IGO 3.0](https://licensebuttons.net/l/by-nc/3.0/igo/80x15.png)](https://creativecommons.org/licenses/by/3.0/igo)
-![CI Build](https://img.shields.io/github/actions/workflow/status/WorldHealthOrganization/smart-trust/ghbuild.yml)  
+![CI Build](https://img.shields.io/github/actions/workflow/status/costateixeira/smart-trust/ghbuild.yml)
```
Comment (ritikarawlani) [inline README.md:3]: @costateixeira is this change intended to refer to your repo? Pls review the Readme file changes

File: sushi-config.yaml [hunk 2]
```diff
@@ -30,3 +30,5 @@ dependencies:
     who.ddcc: dev
     who.ips-pilgrimage: dev
     
+# parameters:
+#   apply-version: true
```
Comment (ritikarawlani) [inline sushi-config.yaml:34]: @costateixeira  please also upgrade the version to 1.1.1 with this change and describe change in publication-request json

**Review by ritikarawlani — COMMENTED** [review]
@costateixeira  please resolve comments

**Comment by litlfred** [comment]
anything to do here @costateixeira  @ritikarawlani ?

**Review by costateixeira — COMMENTED** [review]
remove commented out text
